radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/openapitools/model/HealthResponse.java

```diff
@@ -153,29 +153,29 @@ public void setNetworkStatus(NetworkStatusEnum networkStatus) {
   }
 
 
-  public HealthResponse currentForkHash(String currentForkHash) {
-    this.currentForkHash = currentForkHash;
+  public HealthResponse currentForkName(String currentForkName) {
+    this.currentForkName = currentForkName;
     return this;
   }
 
   /**
-   * Get currentForkHash
-   * @return currentForkHash
+   * Get currentForkName
+   * @return currentForkName
    **/
   @javax.annotation.Nonnull
   @ApiModelProperty(required = true, value = "")
-  @JsonProperty(JSON_PROPERTY_CURRENT_FORK_HASH)
+  @JsonProperty(JSON_PROPERTY_CURRENT_FORK_NAME)
```

Comment:
I see that sometimes the "hash" suffix has been replaced by "name" and sometimes it  has just been removed, like at line 239.

Reply:
that's because in some cases it's just a name (string) and in other cases it's an object with a name and other fields

Reply:
Does that mean that `HealthResponseUnknownReportedForksHashes` shouldn't be named like that before and the fix was just to remove the suffix?

Reply:
yeah, it should be `HealthResponseUnknownReportedForks` from the beginning